packages/backend-common/src/dynamodb.ts

```diff
@@ -39,6 +39,7 @@ export const TranscriptionDynamoItem = z.object({
 	userEmail: z.string(),
 	completedAt: z.optional(z.string()), // dynamodb can't handle dates so we need to use an ISO date
 	isTranslation: z.boolean(),
+	languageCode: z.optional(z.string()),
```

Comment:
Is this an iso country code? If so this type could be more narrow if we use our [`LanguageCode`](https://github.com/guardian/transcription-service/blob/71f91751872589d9c02e11112239e1bd1de6d80c/packages/common/src/languages.ts#L111) type